deployment/ansible/roles/raster-foundry.jenkins/templates/jenkins.conf.j2

```diff
@@ -0,0 +1,18 @@
+server {
+    listen 80;
+    server_name {{ jenkins_server_name }};
+
+    location / {
+
+      proxy_set_header        Host $host:$server_port;
+      proxy_set_header        X-Real-IP $remote_addr;
+      proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header        X-Forwarded-Proto $scheme;
+
+      # Fix the "It appears that your reverse proxy set up is broken" error.
```

Comment:
I don't think this comment adds much value here. Looks like it may have come from a Jenkins example for setting up Nginx as a reverse proxy.